src/modules/commander/state_machine_helper.cpp

```diff
@@ -691,7 +691,7 @@ bool set_nav_state(vehicle_status_s *status, actuator_armed_s *armed, commander_
 					status->nav_state = vehicle_status_s::NAVIGATION_STATE_AUTO_LOITER;
 
 				} else if (status_flags.condition_local_altitude_valid) {
-					if (status->is_rotary_wing) {
+					if (status->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING) {
```

Comment:
At some point we'll have to look into the failsafes for rovers, so how things are handled if the `vehicle_type` is neither `ROTARY_WING` nor `FIXED_WING`.

However, that's probably going into a future pull request.